core/models.py

```diff
@@ -6,10 +6,15 @@ class Deck(models.Model):
     description = models.CharField(max_length=1200, verbose_name="Description of deck")
     tag = models.CharField(max_length=50, verbose_name="tag")
     flashCard = models.ManyToManyField('FlashCard', related_name='flashCard', through='DeckFlashCard')
+    public = models.BooleanField(default=False)
+    owner = models.ForeignKey("auth.User", related_name="owner", on_delete=models.CASCADE, null=True)
```

Comment:
As I understand this response: https://stackoverflow.com/questions/2642613/what-is-related-name-used-for-in-django
related_name in this situation supposed to be deck

Reply:
Hmm and what do you thing about other related_name in foreign keys in models.py file?

Reply:
ok now i understand

Reply:
changed to deck